lib/generators/geoblacklight/install_generator.rb

```diff
@@ -110,21 +110,37 @@ def copy_package_json
       copy_file "package.json", "package.json"
     end
 
-    # Vite - Config files
-    def copy_config_vite_json
-      copy_file "vite.json", "config/vite.json"
-      copy_file "vite.config.ts", "vite.config.ts"
-      run "yarn install"
-    end
-
+    # Pick a version of the frontend asset package and install it.
     def add_frontend
+      # If in local development or CI, install the version we made linkable in
+      # the test app generator. This will make it so changes made in the outer
+      # directory are picked up automatically, like a symlink. Note that this
+      # does NOT install the dependencies of the package, so we have to make
+      # sure to install those separately. See:
+      # https://classic.yarnpkg.com/lang/en/docs/cli/link/
```

Comment:
This link is possibly stale, but Yarn 2's docs on link are so much worst than Yarn 1.